packages/components/cypress/e2e/popover.cy.ts

```diff
@@ -43,13 +67,13 @@ describe('popover', { baseUrl: null, includeShadowDom: true }, () => {
         $trigger.html(`<span class="nested-element">${originalText}</span>`);
       });
 
-      cy.get('@popover').should('not.be.visible');
+      cy.get('@popover-content').should('not.be.visible');
       cy.get('@trigger').should('have.attr', 'aria-expanded', 'false');
       cy.get('.nested-element').click();
-      cy.get('@popover').should('be.visible');
+      cy.get('@popover-content').should('be.visible');
       cy.get('@trigger').should('have.attr', 'aria-expanded', 'true');
-      cy.get('post-closebutton').click();
-      cy.get('@popover').should('not.be.visible');
+      cy.get('@post-popover').find('post-closebutton').click();
```

Comment:
Instead of doing that multiple times, you could define the post-closebutton as an alias in the `beforeEach` function above and then directly use it in the tests when needed.

`cy.get('@post-popover').find('post-closebutton').as('closeButton');`